### ISSUE TITLE
Choose emulator models

### ISSUE THREAD

**Issue by mastoffel** [alan-turing-institute/autoemulate#3]
Choose a small-ish (~6-10) number of models that work well with a variety of regression type problems. 

Get models so that emulation works well with:
* regression type problems with multiple continuous inputs and multiple continuous outputs
* small to medium sized datasets (say 50 - 10000 datapoints)

Packages:
* [mogp-emulator](https://github.com/alan-turing-institute/mogp-emulator) for GPs and Multi-Output GPs
* [smt](https://github.com/SMTorg/smt) for a range of emulators
* [Surrogates.jl](https://docs.sciml.ai/Surrogates/latest/) for inspiration from Julia
* [scikit-learn](https://scikit-learn.org/stable/index.html)

Models:
- [x] Gaussian Processes
- [x] Random Forests
- [x] Radial Basis Functions
- [x] Second Order Polynomials
- [x] Gradient Boosting
- [x] MLP
- [x] #80
- [x] SupportVectorMachines

**Comment by mastoffel** [comment]
After discussing with Eric and Steve, GPs and Neural Networks are the place to start.

**Comment by mastoffel** [comment]
So far we've got:

* GPs
* Random Forest
* simple MLP

On the list:
* Neural Processes
* Gaussian Process Regression Trees

**Comment by kallewesterling** [comment]
@mastoffel We need to chat about this issue as this seems like an ongoing activity that we might want to separate out as such. I.e. there's no endpoint when we can say that we're _done choosing_ the emulator models.

**Comment by mastoffel** [comment]
> @mastoffel We need to chat about this issue as this seems like an ongoing activity that we might want to separate out as such. I.e. there's no endpoint when we can say that we're _done choosing_ the emulator models.

Yes, I think this should be closed. From now on we only add / delete specific models.
